addons/airway/functions/fnc_treatmentAdvanced_airwayLocal.sqf

```diff
@@ -35,6 +35,14 @@ if (_patient getVariable [QGVAR(airway_item), ""] isEqualTo "Guedeltubus" && _us
     [_medic, _usedItem] call ACEFUNC(common,addToInventory);
 };
 
+if (_usedItem == "kat_larynx") then {
+    _patient setVariable [QGVAR(kingLTisSet), true, true];
+};
+
+if (_usedItem == "kat_guedel") then {
+    _patient setVariable [QGVAR(guedeltubusisSet), true, true];
+};
+
 _patient setVariable [QGVAR(airway), true, true];
 _patient setVariable [QGVAR(obstruction), false, true];
 _patient setVariable [QGVAR(airway_item), _classname, true];
```

Comment:
We already set the classname of the used item, why do we need another variable for it?

Reply:
Heda likes to do stuff "different way"